docs/inventory.md

```diff
@@ -135,7 +144,8 @@ Now you can set the path of the ansible inventory in the source:
 [Netbox](https://netbox.readthedocs.io/en/stable/) is often used to store devices type, management address and other useful information to be used in network automation. Suzieq can pull device data from Netbox selecting them by tag (currently only one per each source).
 To do so a token to access the netbox API is required as well as the netbox instance url.
 
-The token can be directly written in the inventory or it can be stored in an environment variable and indicated using the format `env:<ENV_TOKEN>`.
+The token is considered a [sensible data](#sensible-data), so it can be specified via an environment variable.
+The input via the stdin is not supported.
```

Comment:
I liked the statement about "env: <ENV_TOKEN>". Why did you take it out?

Reply:
I thought it was a repetition since I wrote that it is a sensible data, but I added it back. Writing the same thing twice sometimes is better